src/probnum/problems/zoo/filtsmooth/_ssm_examples.py

```diff
@@ -0,0 +1,328 @@
+import numpy as np
+
+from probnum import diffeq, filtsmooth, problems, randvars, statespace
+
+__all__ = ["car_tracking", "logistic_ode", "ornstein_uhlenbeck", "pendulum"]
+
+
+def car_tracking(delta_t=0.2, measvar=0.5, t_max=20.0):
+    r"""State space model for a simple car-tracking scenario.
```

Comment:
this got me thinking... should this actually be a state space model zoo, not a filtsmooth model zoo?

Reply:
Hm, true. I don't have a strong opinion on this, but would also tend to `statespace`. Shall I change it?